tests/testData/opd/opd.zer

```diff
@@ -1,19 +1,20 @@
-# The followings are OPD in rotation angle of -0.00 degree in nm from z4 to z22:
-191: [-4.15577837e+02 4.03053050e+02  6.19438978e+02 -4.44665578e+00
- 1.00741331e+01  3.55263409e+01  -9.61023230e+00  1.44900235e-01
- -1.02512184e+00  -3.99302251e+01  9.81672032e+00 1.09511113e+02
-  -1.01009395e+00 -1.03280016e+00 2.30629852e+00  2.55961095e+01
-  -1.29456402e+00  3.40161544e+01 -1.13822260e+01]
-195: [-407.96514385   -69.61576656  631.04232073   50.49126701   7.97330041
-  252.35981927  30.87974042   -3.93741576   17.25916179  25.49876989
-   19.17288935 113.36121766   2.97981464   -6.73377281   6.2956465
-  -31.79557922   -14.85384164   29.62548445  -10.81317459]
-199: [-406.87545247   -43.41883523  678.05355873    3.63692016   -43.14243043
-   22.29089894 180.82275562   -5.65229654  -21.64993839  22.26986463
-    3.69490561 103.41585077   7.4490198     3.37207099   -42.23624375
-   17.04691       -4.30762799   23.83670381  -10.49902789]
-203: [-3.89941394e+02 2.85791837e+02  6.41296675e+02  3.26727427e+01
-  -3.35857590e+01  1.90287712e+02 1.75210163e+02 -1.06136662e+01
- -1.22732198e+01  -2.09839551e+01  4.17355247e-01 1.08153979e+02
- 5.16252005e+00 -1.42877583e+00  -3.38332003e+01 -2.19601267e+01
-  -5.94293965e+00  3.59018468e+01 -8.63500688e+00]
+# The followings are OPD in rotation angle of 0.00 degree in nm from z4 to z22:
```

Comment:
Should these not go to 28 now?

Reply:
Why did these change if you didn't change them to go to 28?

Reply:
This change doesn't affect which Zks are stored, but which are fitted (i.e. previously we were fitting  zk1:22, selecting only 4:22,  and now we're fitting zk1:28,  selecting again 4:22 since that's what the rest of the pipeline assumes).